src/components/Nav.js

```diff
@@ -5,29 +5,53 @@ import {SubtleLink} from "../styles/Typography";
 import {AppNav, HomeNav} from "../styles/Header";
 import {SpaceBetween} from "../styles/Grid";
 import {Link} from "react-router-dom";
+import ProfileAvatar from "../styles/ProfileAvatar";
+import Octicon, {getIconByName} from "@primer/octicons-react";
 
 function LeftSide({user, handleLogIn, handleLogOut}) {
   return (
     <FloatLeft>
+      <Link style={{verticalAlign: "middle"}} to="/">
+        <img alt="open sauced" src={logo} />
+      </Link>
       <ul>
         <li>
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("repo")} />
+          </span>
           <SubtleLink href="https://dev.to/t/opensauced">Blog</SubtleLink>
         </li>
         <li>
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("mark-github")} />
+          </span>
```

Comment:
This does not align with GitHub's styleguide. https://github.com/logos

The octomark and the name GitHub are not meant to appear next to each other.